README.txt

```diff
@@ -0,0 +1,46 @@
+### Specifications for the WetlandElevationChangeTable.csv ###
+
+FileName
+########
+Type - Text
+Null - False, cannot contain null values
+Desc - Field contains the names of each input point feature
+		class or shapefile. Names must exactly match the file
+		name, and will fail if aliases are supplied. Do not supply the file extension. One row represents one input file. 
+
+
+Watershed
+#########
+Type - Text
+Null - False, cannot contain null values
+Desc - Name of the watershed. Watersheds are one part of 
+		organizing the analysis groups, so that locations can be evaluated over a period of years.
+
+Year
+####
+Type - Date/year
+Null - False, cannot contain null values
+Desc - 4-digit year (ex. 2002, 1909, 2016)
```

Comment:
seems like it could more relaxed - like some arbitrary string (without spaces hopefully). Maybe should be clear about whether you will handle datasets that have the same group and year.